http/handler.go

```diff
@@ -68,34 +83,23 @@ func WithPprofEnabled(enabled bool) HandlerOptFn {
 	}
 }
 
-func WithHealthHandler(h http.Handler) HandlerOptFn {
```

Comment:
Why are we dropping the `health` and `ready` handler code? They're just not called?

Reply:
Yes, unused.